.github/PULL_REQUEST_TEMPLATE/small-change.md

```diff
@@ -2,6 +2,13 @@
 
 <!-- Write a brief description of this PR. -->
 
+## Tests performed
+
+<!-- Describe how you have tested this PR. -->
+<!-- Must fill in feat (feature), fix (bug fix), perf (performance), and any other PRs that should be tested. -->
```

Comment:
How about this? I feel it's clearer than `any other PRs that should be tested`.
Also, I think this should be written also in `standard-change.md`.

```suggestion
<!-- Must fill in feat (feature), fix (bug fix), perf (performance), and any other non-trivial PRs. -->
```

Reply:
I think non-trivial PRs use standard template not small template.

Reply:
Hmm, I thought you'd like to write the same content in both templates, but I misunderstood something.
So, could you describe the background and your purpose?
https://github.com/autowarefoundation/autoware/pull/3389#issuecomment-1492844394

Reply:
Ok, I updated the backgound in the PR description.

Reply:
Thank you!
Then, how about this?

```suggestion
<!-- Although the default value is set to "Not Applicable.", please complete this section if the type is either [feat, fix, perf], or if requested by the reviewers. -->
```

Reply:
Sorry, maybe complete -> update would be better here? :thinking: 
I'll update the suggestion above.

Reply:
Perfects!!! Easier to understand.